lib/pf/Switch/Cisco/WLC.pm

```diff
@@ -429,10 +411,10 @@ sub returnRadiusAccessAccept {
     my @av_pairs = defined($radius_reply_ref->{'Cisco-AVPair'}) ? @{$radius_reply_ref->{'Cisco-AVPair'}} : ();
 
     my $role = $self->getRoleByName($args->{'user_role'});
-    if ( isenabled($self->{_UrlMap}) && $self->supportsUrlBasedEnforcement ) {
+    if ( isenabled($self->{_UrlMap}) && $self->externalPortalEnforcement ) {
         if ( defined($args->{'user_role'}) && $args->{'user_role'} ne "" && defined($self->getUrlByName($args->{'user_role'}) ) ) {
             my $redirect_url = $self->getUrlByName($args->{'user_role'});
-            $args->{'session_id'} = "cep".$self->setSession($args) if ($redirect_url =~ /\$session_id/);
+            $args->{'session_id'} = "sid".$self->setSession($args) if ($redirect_url =~ /\$session_id/);
```

Comment:
"cep" originally stands for "cisco external portal". "sid" stands for "session id" which is more generic if it comes that we support other vendors that uses session based external portal flow.